<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveBlocksWidgetBase</class>
 <widget class="QWidget" name="SaveBlocksWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>152</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="chkSelectionOnly">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>If checked, &lt;b&gt;save only the blocks that overlap with the current selection&lt;/b&gt;, otherwise the whole file. &lt;br&gt;&lt;i&gt;Please note that this option is disabled if nothing is selected or the selection already covers the whole file.&lt;/i&gt;</string>
     </property>
     <property name="text">
      <string>save selection only</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblNumbering">
     <property name="text">
      <string>numbering:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbNumbering">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>determines how the &lt;b&gt;numbering of the files&lt;/b&gt; should happen:&lt;ul&gt;
&lt;li&gt;continue after the &lt;b&gt;highest index&lt;/b&gt; that can be found, avoiding collissions with existing files&lt;/li&gt;
&lt;li&gt;always start numbering from &lt;b&gt;one&lt;/b&gt;, possibly colliding with already existing files&lt;/li&gt;
&lt;/ul&gt;</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>continue after highest index</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>always start with 1, overwrite if necessary</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtExample">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>example of the first file name that will be saved, using the file name pattern above and the numbering below</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblExample">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>example:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbPattern">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>&lt;p&gt;Select the pattern that is used to create the file name of each saved block. It currently understands the following placeholders:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;&lt;tt&gt;&lt;b&gt;[%filename]&lt;/b&gt;&lt;/tt&gt;name of the file, without the path&lt;/li&gt;
&lt;li&gt;&lt;tt&gt;&lt;b&gt;[&lt;/b&gt;%&lt;i&gt;0&lt;/i&gt;&lt;/tt&gt;&lt;i&gt;N&lt;/i&gt;&lt;tt&gt;&lt;b&gt;nr]&lt;/b&gt;&lt;/tt&gt;the index of the file, optionally with leading zeroes (if &lt;tt&gt;'0&lt;/tt&gt;' prepended) and optionally a fixed number of digits (&lt;tt&gt;N&lt;/tt&gt;)&lt;/li&gt;
&lt;li&gt;&lt;tt&gt;&lt;b&gt;[%&lt;/b&gt;&lt;i&gt;0&lt;/i&gt;&lt;i&gt;N&lt;/i&gt;&lt;b&gt;count]&lt;/b&gt;&lt;/tt&gt;the number of files that will be saved&lt;/li&gt;
&lt;li&gt;&lt;tt&gt;&lt;b&gt;[%&lt;/b&gt;&lt;i&gt;0&lt;/i&gt;&lt;i&gt;N&lt;/i&gt;&lt;b&gt;total]&lt;/b&gt;&lt;/tt&gt;the highest index within the files that will be saved&lt;/li&gt;
&lt;/ul&gt;</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblPattern">
     <property name="text">
      <string>file name pattern:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="10" margin="10"/>
 <tabstops>
  <tabstop>cbPattern</tabstop>
  <tabstop>txtExample</tabstop>
  <tabstop>cbNumbering</tabstop>
  <tabstop>chkSelectionOnly</tabstop>
 </tabstops>
 <includes>
  <include location="global">klocale.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
